IFC4x3/Sections/Core data schemas/Schemas/IfcProductExtension/Entities/IfcVehicle/Documentation.md

```diff
@@ -0,0 +1 @@
+A vehicle.
```

Comment:
A vehicle is a "non-fixed" or "mobile" transportation device as far as I can understand. The distinction between IfcTransportElement and IfcVehicle should be clear.